### PR TITLE
Translated some missing English sentences to Dutch

### PR DESCRIPTION
**Pull request by mjarkk**

### CODE REVIEW

**Comment by dawidd6** [comment]
@mjarkk Sure, will do soon.

File: pkg/i18n/dutch.go [hunk 1]
```diff
@@ -417,16 +417,16 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: `return to files panel`,
+			Other: `ga terug naar het bestanden paneel`,
 		}, &i18n.Message{
 			ID:    "CantFindHunks",
-			Other: `Could not find any hunks in this patch`,
+			Other: `Kan geen hunks vinden in deze patch`,
 		}, &i18n.Message{
 			ID:    "CantFindHunk",
-			Other: `Could not find hunk`,
+			Other: `kan geen hunk finden`,
```
Comment (glvr182) [inline pkg/i18n/dutch.go]: finden can be replaced with "vinden"

Comment (mjarkk) [reply pkg/i18n/dutch.go]: Thx i'll do that!

File: pkg/i18n/dutch.go [hunk 2]
```diff
@@ -417,16 +417,16 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
-			Other: `return to files panel`,
+			Other: `ga terug naar het bestanden paneel`,
 		}, &i18n.Message{
 			ID:    "CantFindHunks",
-			Other: `Could not find any hunks in this patch`,
+			Other: `Kan geen hunks vinden in deze patch`,
 		}, &i18n.Message{
 			ID:    "CantFindHunk",
-			Other: `Could not find hunk`,
+			Other: `kan geen hunk vinden`,
```
Comment (Glennmen) [inline pkg/i18n/dutch.go]: To keep structure the same `kan` should be capitalized.

Comment (mjarkk) [reply pkg/i18n/dutch.go]: Thanks forgot to capitalize that, I'll change it!

**Pull request merged**